minio/operator
# feat: support file ref resource

support file ref resource

- [x] Support `mc support callhome`

- [x] Support `mc license register`

- [x] Support Keyfile config with `ref(secret|configmap.[name].[key])`

- [x] Support key config with `get(secret|configmap.name.key)` 

how to test:
UT should cover this.
And this is test yaml:
```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: mytestconfig
data:
  client.key:  keykeykeykeykeykeykeykeykey
  client.cert: certcertcertcertcertcert
---
apiVersion: job.min.io/v1alpha1
kind: MinIOJob
metadata:
  name: minio-test-job
spec:
  serviceAccountName: mc-job-sa
  tenant:
    name: mytest-minio
  commands:
    -  op: admin/config/set
       args:
         webhookName: mytestName
         endpoint: https://mytest.com
         client_key: ref(configmap.mytestconfig.client.key)
         client_cert: ref(configmap.mytestconfig.client.cert)
    - name: add-my-policy
      op: admin/policy/create
      args:
        name: memes-access
        policy: |
          {
              "Version": "2012-10-17",
              "Statement": [
                  {
                      "Effect": "Allow",
                      "Action": [
                          "s3:*"
                      ],
                      "Resource": [
                          "arn:aws:s3:::memes",
                          "arn:aws:s3:::memes/*"
                      ]
                  }
              ]
          }
```
